src/app/Dashboard/AutomatedAnalysis/ClickableAutomatedAnalysisLabel.tsx

```diff
@@ -42,18 +42,19 @@ import { InfoCircleIcon } from '@patternfly/react-icons';
 import { css } from '@patternfly/react-styles';
 import popoverStyles from '@patternfly/react-styles/css/components/Popover/popover';
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
 export interface ClickableAutomatedAnalysisLabelProps {
   label: RuleEvaluation;
-  isSelected: boolean;
 }
 
 export const clickableAutomatedAnalysisKey = 'clickable-automated-analysis-label';
 
 export const ClickableAutomatedAnalysisLabel: React.FunctionComponent<ClickableAutomatedAnalysisLabelProps> = ({
   label,
-  isSelected,
 }) => {
+  const { t } = useTranslation();
```

Comment:
I think either way is fine but maybe we can have to consistent to either use:

```typescript
const { t } = useTranslation();
```

or 

```typescript
const [t] = useTranslation();
```

Reply:
Yeah I have been flipping between the two. I think I prefer the first over the latter, but what do you think?

Reply:
Same here! I think we can go with the first one then :D